Dockerfile

```diff
@@ -12,7 +12,7 @@ ARG DOCKERCLI_VERSION=v27.3.1
 # cli version used for integration-cli tests
 ARG DOCKERCLI_INTEGRATION_REPOSITORY="https://github.com/docker/cli.git"
 ARG DOCKERCLI_INTEGRATION_VERSION=v17.06.2-ce
-ARG BUILDX_VERSION=0.18.0
+ARG BUILDX_VERSION=v0.18.0
```

Comment:
I don't think this change is correct; we use this version to install a version of buildx to used in our tests, and for that we use the `docker/buildx-bin` image, which uses a tagging scheme without `v` prefix for the version;
https://hub.docker.com/r/docker/buildx-bin/tags?name=0.19.2


It's a bit confusing though because buildx's release _branches_ do use a `v` prefix, which get also built, and now those do have a v-prefix 😓.
https://hub.docker.com/r/docker/buildx-bin/tags?name=v0.19


https://github.com/moby/moby/blob/1e033e2d983df0f20caa8da5320dee3c3c2d7d63/Dockerfile#L453

Reply:
@crazy-max I wonder if we should consider aligning on those release-branch names; I was even considering to use the containerd scheming (`release/xxxx` prefix); I know we deliberately chose branches without a `v` prefix in moby and docker/cli to prevent them from being confused for tags (there was a bug in go modules related to that choice that happily showed a branch name as tag in go modules 😬)

Reply:
I did a script to automate some of my custom-builds of docker, but then I realized this problem specifically with this version `0.18.0` (which broke my script) since all the other versions do have the 'v' prefix.